src/location-distance-utils.c

```diff
@@ -21,31 +21,33 @@
 
 #include "location-distance-utils.h"
 
-static double d2r(double deg)
-{
-	return (deg * M_PI / 180);
-}
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
+
+// scaling factor from degrees to meters at equator
+// == DEG_TO_RAD * radius of earth
+static float LOCATION_SCALING_FACTOR = 111318.84502145034f;
+
+#define MAX(a, b) (((a) > (b)) ? (a) : (b))
+
+#define D2R (M_PI / 180.0f)
+// #define R2D (180.0f / M_PI)
 
-static double r2d(double rad)
+// compensate for shrinking longitude towards poles
+float lng_scale(double lat)
```

Comment:
static? also, why float and not double?

Reply:
Ah! it's my bad, it should be double,`dist` type was float in my head for some reason.
I'm not sure what you mean by, the original `r2d` was static.